androidApp/src/main/java/com/mbta/tid/mbta_app/android/nearbyTransit/NearbyStopView.kt

```diff
@@ -72,7 +72,7 @@ fun NearbyStopView(
                             patternsAtStop.elevatorAlerts.size,
                             patternsAtStop.elevatorAlerts.size
                         ),
-                    modifier = Modifier.placeholderIfLoading().alpha(0.6f),
+                    modifier = Modifier.placeholderIfLoading().alpha(0.6f).weight(.4f),
```

Comment:
This looked good at wonderland but not at Davis, seeing if min intrinsic width will help here too

Reply:
This is the best I have for davis right now. Confirmed [w/ product](https://mbta.slack.com/archives/C05QMG9GS9M/p1740179835146629?thread_ts=1740174183.461249&cid=C05QMG9GS9M) that it is OK for now.

![image](https://github.com/user-attachments/assets/dbbe721f-67fa-45c4-96b9-3537bbb38176)